client/controller/api/apps.go

```diff
@@ -6,7 +6,7 @@ type App struct {
 	ID      string `json:"id"`
 	Owner   string `json:"owner"`
 	Updated string `json:"updated"`
-	URL     string `json:"url"`
+	URL     string
```

Comment:
the tl;dr is that since some of the struct fields have tags and `URL` doesn't, the [marshaler rules](https://godoc.org/encoding/json#Marshal) state that `URL` will be ignored.

Here's the longer version, from the docs. The first rule below is in action here:

```
The Go visibility rules for struct fields are amended for JSON when deciding which field to marshal or unmarshal. If there are multiple fields at the same level, and that level is the least nested (and would therefore be the nesting level selected by the usual Go rules), the following extra rules apply:

1) Of those fields, if any are JSON-tagged, only tagged fields are considered, even if there are multiple untagged fields that would otherwise conflict. 2) If there is exactly one field (tagged or not according to the first rule), that is selected. 3) Otherwise there are multiple fields, and all are ignored; no error occurs.
```